exercise10_2.py

```diff
@@ -31,8 +31,8 @@
 """
 
 
-dictionary_hours = dict()               # Initialize variables
-lst = list()
+Mydictionary_hours = dict()               # Initialize variables
```

Comment:
Typically only uppercase characters are reserved for classes and consts, among some other special types. So, so snake_case here is more appropriate.